examples/meta/src/gaussian_process/classifier.sg.in

```diff
@@ -6,20 +6,21 @@ CSVFile f_labels_test("@SHOGUN_DATA@/classifier_4class_2d_linear_labels_test.dat
 #![create_features]
 Features features_train = features(f_feats_train)
 Features features_test = features(f_feats_test)
-MulticlassLabels labels_train(f_labels_train)
-MulticlassLabels labels_test(f_labels_test)
+
+Labels labels_train = labels(f_labels_train)
+Labels labels_test = labels(f_labels_test)
 #![create_features]
 
 #![create_appropriate_kernel_and_mean_function]
 Kernel k = kernel("GaussianKernel", log_width=2.0)
-ConstMean mean_function()
+MeanFunction mean_function = gp_mean("ConstMean")
 #![create_appropriate_kernel_and_mean_function]
 
 #![create_instance]
-LikelihoodModel gauss_likelihood = gp_likelihood("SoftMaxLikelihood")
-gauss_likelihood.put("seed", 1)
-MultiLaplaceInferenceMethod inference_method(k, features_train, mean_function, labels_train, gauss_likelihood)
-GaussianProcessClassification gp_classifier(inference_method)
+LikelihoodModel gauss_likelihood = gp_likelihood("SoftMaxLikelihood", seed=1)
```

Comment:
@karlnapf shoudn't we be passing the seed to the GP instead?

Reply:
I think that would be better, yes